adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -854,6 +855,36 @@ public void export(final UnusableTripNotice toExport) throws IOException {
                 .writeTo(streamGenerator.getStream());
     }
 
+    @Override
+    public void export(final DecreasingStopTimeDistanceErrorNotice toExport) throws IOException {
+        decreasingStopTimeDistanceToProto(GtfsValidationOutputProto.GtfsProblem.Severity.ERROR, toExport);
+
+    }
+
+    @Override
+    public void export(final DecreasingStopTimeDistanceWarningNotice toExport) throws IOException {
+        decreasingStopTimeDistanceToProto(GtfsValidationOutputProto.GtfsProblem.Severity.WARNING, toExport);
+    }
+
+    private void decreasingStopTimeDistanceToProto(final GtfsValidationOutputProto.GtfsProblem.Severity severity,
+                                                   final Notice toExport)
+            throws IOException {
```

Comment:
Could be moved to above line

Reply:
Since the warning notice has been removed, this method is no more used.